tools/check_test.js

```diff
@@ -30,6 +30,12 @@ function Driver() {
     fail: 0,
     skip: 0,
     timeout: 0,
+    json: {
+      bin: { text:0, total:0, data:0, bss:0, rodata:0 },
```

Comment:
please read: https://github.com/Samsung/iotjs-test-results/blob/gh-pages/js/render_entries.js#L56
It is used there. For now it is not filled with data, my guess is it will be filled with https://github.com/rtakacs/remote-testrunner or similar solution. I am not sure IoT.js itself has this data.